Anytype/Sources/ApplicationLayer/Launch/LaunchCircle.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+import SwiftUI
+
+struct LaunchCircle: View {
+    
+    private enum Constants {
+        static let maxSide: CGFloat = 128
+        static let middleSide: CGFloat = 48
+        static let minSide: CGFloat = 16
+        static let duration: CGFloat = 1.6
+        static let initialDuration: CGFloat = Constants.duration + Constants.duration
```

Comment:
Shouldn't we make it relative depending on the device size? On iPad it will be tiny for example?

Reply:
interesting idea, I'll think about it with the designer